plugins/modules/provision_workflow_manager.py

```diff
@@ -436,6 +485,52 @@ def get_execution_status(self, execution_id=None):
         self.result.update(dict(assignment_task=response))
         return result
 
+    def get_execution_status_wireless(self, execution_id=None):
+        """
+        Fetches the status of the BAPI once site wireless provision API is called
+
+        Parameters:
+          - self: The instance of the class containing the 'config' attribute
+                  to be validated.
+          - execution_id: execution_id of the BAPI API.
+        Returns:
+          The method returns the status of the BAPI used to track wireless provisioning.
+          Returns True if the status is not failed, otheriwse returns False.
+        Example:
+          Post creation of the provision task, this method fetheches the task
+          status.
+
+        """
+        result = False
+        params = {"execution_id": execution_id}
+        while True:
+            response = self.dnac_apply['exec'](
+                family="task",
+                function="get_business_api_execution_details",
+                params=params,
+                op_modifies=True
+            )
+            self.log("Response collected from 'get_business_api_execution_details' API is {0}".format(str(response)), "DEBUG")
+            self.log("Execution status for the execution id {0} is {1}".format(str(execution_id), str(response.get("status"))), "INFO")
+            if response.get('bapiError') or response.get("status") == "FAILURE":
+                if response.get("bapiError") == "Device was already provisioned , please use provision update API to reprovision the device":
```

Comment:
usually we won't leave space before comma..  Do we get bapiError like this? Can you check?

Reply:
Yes we have a bapi Error like this